.gitignore

```diff
@@ -3,6 +3,7 @@
 # dependencies
 /node_modules
 /.pnp
+.env
```

Comment:
Why ignoring `.env`? 
(reminder: it should not contain secrets in any case)
We might want you use `.env` files in the future so - either include it in the PR or simply don't commit the file